lib/index.js

```diff
@@ -126,11 +126,10 @@ function* createEverything() {
 
   var adminTestDatabaseClient = pgp(adminTestDatabase);
   yield adminTestDatabaseClient.task(alterPrivileges);
-
-  pgp.end();
```

Comment:
I don't think there are any advantages to the connection pool approach, given the nature of the utility, so maybe that makes more sense. I'll give it a shot and see how it turns out.